eventParser/eventParser.go

```diff
@@ -103,13 +106,9 @@ func NewEventsEndpoint(p *EventParser) endpoint.Endpoint {
 
 		select {
 		case p.requestQueue.Load().(chan *wrp.Message) <- message:
-			if p.measures != nil {
```

Comment:
if you're going to remove these checks, you have to check that the measures aren't nil in the creation of the event parser object.